graf2d/fitsio/inc/TFITS.h

```diff
@@ -43,10 +44,11 @@ class TFITSHDU : public TNamed {
       kTableHDU
    };
 
-   enum EColumnTypes {     // Column data types
-      kRealNumber,
-      kString,
-      kRealVector
+   enum EColumnTypes { // Column data types
+      kString,         // the column contains a string
+      kRealNumber,     // the column contains a scalar
+      kRealArray,      // the column contains a fixed-length array
+      kRealVector      // the column contains a variable-length array
    };
```

Comment:
Here I try to use a nomenclature similar to C++ basic objects: a `kRealArray` is an array, i.e. an object of fixed length, a `kRealVector` is a vector, i.e. an object whose length is not fixed. Hope you approve it, the following checks are based on it.